packages/components/src/TCRUserDashboard/DashboardNewsroom.tsx

```diff
@@ -155,55 +166,96 @@ const DashboardNewsroomBase: React.FunctionComponent<DashboardNewsroomProps> = p
+        <NewsroomProceeds newsroomAddress={props.newsroomAddress} boostType={"externallink"} />
+        <StyledDashboardNewsroomBorder />
+        <StyledDashboardNewsroomSubHdr>Project Boosts</StyledDashboardNewsroomSubHdr>
+        <p>
+          Project Boosts lets your newsroom host one-off fundraisers anywhere on your website to fund and produce larger
+          journalism projects. Share your vision and how the money will be used so your readers can get behind it.
+        </p>
+        <p>
+          You can embed Projects Boosts on your site as well. You just need to copy the embed from each Project Boost.{" "}
           <a href={urlConstants.FAQ_BOOSTS} target="_blank">
             Learn more
           </a>
         </p>
-
-        {/*@HACK We need to include `NewsroomWithdraw` from `sdk` package, but this component is in `components` package which `sdk` uses so we'd have a circular dependency. @TODO/tobek all these TCR dashboard components should be moved into `dapp` package.*/}
-        {props.boostProceeds}
-
         {/*@TODO Because we're in components we can't access dapp routes so we have to hard code the route*/}
         <p>
-          <Button size={buttonSizes.MEDIUM_WIDE} to={`/manage-newsroom/${props.newsroomAddress}/launch-boost`}>
-            Launch New Boost
-          </Button>
+          <InvertedButton size={buttonSizes.MEDIUM_WIDE} to={`/manage-newsroom/${props.newsroomAddress}/launch-boost`}>
+            Launch a New Project Boost
+          </InvertedButton>
         </p>
         <p>
-          {/*@TODO Ideally we could link directly to that tab, see CIVIL-1021*/}
-          View your current Boosts on the "Boosts" tab on{" "}
-          <Link to={props.listingDetailURL}>your newsroom's Registry listing</Link>.
+          <InvertedButton
+            size={buttonSizes.MEDIUM_WIDE}
+            to={`${props.listingDetailURL}/${props.newsroomAddress}/boosts`}
+          >
+            View your Project Boosts
+          </InvertedButton>
         </p>
+        <StyledDashboardNewsroomTerHdr>Proceeds collected from Project Boosts</StyledDashboardNewsroomTerHdr>
+        <NewsroomProceeds newsroomAddress={props.newsroomAddress} boostType={"boost"} />
       </StyledDashboardNewsroomSection>
-
-      <FeatureFlag feature={"stripe-admin"}>
-        <StyledDashboardNewsroomSection>
+      <StyledDashboardNewsroomSection>
+        <StyledDashboardNewsroomHdr>Payments</StyledDashboardNewsroomHdr>
+        {/*@HACK We need to include `NewsroomWithdraw` from `sdk` package, but this component is in `components` package which `sdk` uses so we'd have a circular dependency. @TODO/tobek all these TCR dashboard components should be moved into `dapp` package.*/}
+        {props.boostProceeds}
+        <StyledDashboardNewsroomBorder />
+        <StyledDashboardNewsroomSubHdr>Set Up Credit Card Proceeds</StyledDashboardNewsroomSubHdr>
```

Comment:
Not sure if this copy is from designs but not sure if "proceeds" makes sense here, maybe just "Set Up Credit Card Payments" or "Set Up Credit Card Support"?